AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/CampaignAdminControllerTests.cs

```diff
@@ -23,21 +23,40 @@ public class CampaignAdminControllerTests : InMemoryContextTest
         //delete this line when all unit tests using it have been completed
         private readonly Task taskFromResultZero = Task.FromResult(0);
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact()]
         public void IndexSendsCampaignListQueryWithCorrectDataWhenUserIsOrgAdmin()
         {
+            Organization aginAware = AgincourtAwarenessExtended(Context);
+            Organization pwPrepares = ParkwoodsPreparesExtended(Context);
```

Comment:
I believe those lines are needed because they insert some organizations and campaigns into the context. The test then tries to see if only the campaigns that match the organization are returned.